protocol/superchainerc20/L1-native-factory.md

```diff
@@ -82,32 +82,32 @@ Will follow the [`Proxy.sol` implementation](https://github.com/ethereum-optimis
 **Factory Implementation**
 
 ```solidity
-contract SuperchainERC20Factory is Semver {
-  mapping(address => address) public deployments;
+contract OptimismSuperchainERC20Factory is Semver {
+  mapping(address superc20 => address remoteToken) public deployments;
 
-  event SuperchainERC20Deployed(address indexed superchainERC20, address indexed remoteToken, string name, string symbol, uint8 decimals);
+  event OptimismSuperchainERC20Deployed(address indexed superchainERC20, address indexed remoteToken, string name, string symbol, uint8 decimals);
```

Comment:
We’re looking for ways to establish common standards regarding deployments. For `SuperchainERC20` backed by an L2 native token, we’re considering using `chainId` as an additional input. Should we do the same for L1 bridged tokens (e.g., Ethereum `chainId` as 1)?

The same reasoning can be applied to parameters stored for each representation, based on having a `remoteToken`, `chainId`, and bridge.

Reply:
I think we should focus specifically on L1 native tokens for this factory as they are a majority of the liquidity and then return to the problem of making a more generic factory in the future

Reply:
Follow up context: https://www.notion.so/defi-wonderland/Extending-SuperchainERC20-to-L2-native-tokens-and-custom-e91ca08dd3eb4434834bf74f56237abf